backend/src/main/resources/schema.graphqls

```diff
@@ -111,15 +111,41 @@ type User {
 
 type Query {
   deviceType: [DeviceType]
-  patients: [Patient]
+  patients(facilityId: String): [Patient]
   patient(id: String!): Patient
   organization: Organization
-  queue: [TestOrder]
-  testResults: [TestResult]
+  queue(facilityId: String!): [TestOrder]
+  testResults(facilityId: String!): [TestResult]
   whoami: User!
 }
 type Mutation {
   updateFacility(
+    facilityId: String!
+    testingFacilityName: String!
+    cliaNumber: String
+    street: String
+    streetTwo: String
+    city: String
+    county: String
+    state: String
+    zipCode: String!
+    phone: String
+    orderingProviderFirstName: String!
+    orderingProviderMiddleName: String
+    orderingProviderLastName: String!
+    orderingProviderSuffix: String
+    orderingProviderNPI: String!
+    orderingProviderStreet: String
+    orderingProviderStreetTwo: String
+    orderingProviderCity: String
+    orderingProviderCounty: String
+    orderingProviderState: String
+    orderingProviderZipCode: String!
+    orderingProviderPhone: String
+    deviceTypes: [String]!
+    defaultDevice: String!
+  ): String
```

Comment:
This should return `Facility` (as should the `add` mutation)

Reply:
updated